x-pack/test/cloud_security_posture_functional/page_objects/findings_page.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../ftr_provider_context';
+
+const FINDINGS_ROUTE = 'cloud_security_posture/findings';
+const SECURITY_SOLUTION_APP_NAME = 'securitySolution';
+const EUI_PAGINATION_BUTTON = 'tablePaginationPopoverButton';
+const FINDINGS_NAME_COLUMN_TESTID = 'tableHeaderCell_rule.name_5';
+const getEuiPaginationText = (size: 10 | 25) => `Rows per page: ${size}`;
+const getEuiPaginationRowTestId = (size: 10 | 25) => `tablePagination-${size}-rows`;
+const getEuiPaginationButtonTestId = (index: number) => `pagination-button-${index}`;
```

Comment:
it'd be nicer to get these from `eui`, but not possible for now.  (`tablePaginationPopoverButton` is in `x-pack/test` 12 times)